roles/installer/templates/tower_postgres.yaml.j2

```diff
@@ -60,6 +60,14 @@ spec:
             - name: postgres
               mountPath: '{{ tower_postgres_data_path | dirname }}'
               subPath: '{{ tower_postgres_data_path | dirname | basename }}'
+{% if tower_node_selector %}
```

Comment:
@a-eperez I believe you want to use `tower_postgres_selector` right?

Reply:
Hi, this is indeed a mistake. I have fixed it in 1cc47f7. Thank you.